demo/src/material-ui/app.js

```diff
@@ -6,6 +6,14 @@ import {
   withOpeningDock
 } from "../../../src";
 
+import RaisedButton from "material-ui/RaisedButton";
+import fileDownload from "../file-download";
```

Comment:
What `fileDownload` is for? I don't see it used.

Reply:
My bad, i was using this previously to get the blob from the file. Thank you for pointing this out. Will remove and dev test